aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -135,24 +137,48 @@ When swapping between deployment slots, any system using data protection won't b
 
 For more information, see <xref:security/data-protection/implementation/key-storage-providers>.
 <a name="deploy-aspnet-core-preview-release-to-azure-app-service"></a>
-<!-- revert this after 3.0 supported
-## Deploy ASP.NET Core preview release to Azure App Service
 
-Use one of the following approaches if the app relies on a preview release of .NET Core:
+## Deploy ASP.NET Core 3.0 to Azure App Service
 
-* [Install the preview site extension](#install-the-preview-site-extension).
+ASP.NET Core 3.0 is supported on Azure App Service. If you want to deploy preview releases of .NET Core 3.0+ in between major releases, or in the rare case during a deployment when the runtime is available but the SDK is not yet installed, use one of the techniques below to deploy an ASP.NET Core 3.0 (or later) app. 
+
+* [Specify the .NET Core SDK Version using Azure Pipelines](#specify-the-net-core-sdk-version-using-azure-pipelines)
 * [Deploy a self-contained preview app](#deploy-a-self-contained-preview-app).
 * [Use Docker with Web Apps for containers](#use-docker-with-web-apps-for-containers).
--->
-## Deploy ASP.NET Core 3.0 to Azure App Service
+* [Install the preview site extension](#install-the-preview-site-extension).
 
-We hope to have ASP.NET Core 3.0 available on Azure App Service soon.
+### Specify the .NET Core SDK Version using Azure Pipelines
 
-Use one of the following approaches if the app relies on .NET Core 3.0:
+Using [Azure App Service's CI/CD features](https://docs.microsoft.com/en-us/azure/app-service/deploy-continuous-deployment) makes it easy to set up a continuous integration build with Azure DevOps. Once this build has been created, a few additional tweaks can be made to not only "pin" your SDK version so that you're always confident to build against a consistent .NET Core SDK version. 
```

Comment:
```suggestion
Use [Azure App Service CI/CD scenarios](/azure/app-service/deploy-continuous-deployment) to set up a continuous integration build with Azure DevOps. After the Azure DevOps build is created, optionally configure the build to use a specific SDK version. 
```

btw - Management asked us not to speak of "features" but to try and say "scenarios." It doesn't always work out to be the best choice, so we still use "features" in a few spots.